modules/configuring-your-ide.adoc

```diff
@@ -0,0 +1,47 @@
+:_module-type: PROCEDURE
+
+[id="configuring-your-ide_{context}"]
+= Configuring your IDE
+
+[role='_abstract']
+You can configure some notebook workbenches to get the most out of your data science work.
+
+== Configuring your code-server workbench
+Incorporating standard database clients into your code-server workbench can streamline your workflow, offering a seamless interface for managing databases. You can incorporate different Database Management System (DBMS) clients into your code-server workbench by using built-in extensions. 
+
+For more information on code-server, see link:https://github.com/coder/code-server[code-server in GitHub].
+
+=== Incorporating DBMS clients using extensions
+
+.Prerequisites
+
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you use specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you use specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have created a data science project that has a code-server workbench.
+
+.Procedure
+
+. From the {productname-short} dashboard, click *Data Science Projects*.
++
+The *Data science projects* page opens.
+. Click the name of the project containing the code-server workbench you want to start.
++
+The *Details* page for the project opens.
+. Click the toggle in the *Status* column for the relevant workbench to start a workbench that is not running.
++
+The status of the workbench that you started changes from *Stopped* to *Running*. 
+. After the workbench has started, click *Open* to open the workbench notebook.
+. In the Activity Bar, click the Extensions icon. (image:images/codeserver-extensions-icon.png[The Extensions icon])
+. Search for the name of the extension you want to install. 
+. Click *Install* to add the extension to your code-server environment.
++ 
+The extension you installed appears in the *Browser - Installed* list on the *Extensions* panel.
+
+==== Extensions
+
+See link:https://marketplace.visualstudio.com/vscode[Visual Studio Marketplace] for available third-party extensions that you can consider installing.
```

Comment:
We should link to https://open-vsx.org here.

Reply:
@jiridanek Also - in the IDE, the extensions search says 'Search Extensions in Marketplace'. Does that mean that the results displayed are pulling directly from VS Marketplace, and we should retain the VS link here?

Reply:
Well, there is more than one marketplace. If you recall the code-server faq, we are using the Eclipse marketplace and not the Microsoft one, because of licencing https://github.com/coder/code-server/blob/main/docs/FAQ.md#why-cant-code-server-use-microsofts-extension-marketplace

Reply:
Gotcha. Have made changes in a fresh PR, as this one is already merged. https://github.com/opendatahub-io/opendatahub-documentation/pull/163